docker/rootfs/etc/services.d/go2rtc/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash

set -euo pipefail
```

For safety: https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425?permalink_comment_id=3945021